data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -105,31 +105,22 @@ then
 		--volume "$PWD:/assets:z" \
 		"${CLUSTER_ETCD_OPERATOR_IMAGE}" \
 		/usr/bin/cluster-etcd-operator render \
-		--etcd-ca=/assets/tls/etcd-ca-bundle.crt \
-		--etcd-ca-key=/assets/tls/etcd-signer.key \
-		--manifest-etcd-image="${MACHINE_CONFIG_ETCD_IMAGE}" \
-		--asset-input-dir=/assets/tls \
+		--asset-input-dir=/dev/null/deprecated \
+		--config-output-file=/dev/null/deprecated \
```

Comment:
I made a hash of the deprecation so it will take another ceo PR to remove this options. https://github.com/openshift/cluster-etcd-operator/pull/549 will need to merge first.